pkg/helm/chartproxy/config.go

```diff
@@ -27,7 +27,7 @@ func RegisterFlags(fs *flag.FlagSet) *config {
 
 	cfg := new(config)
 
-	fs.StringVar(&cfg.repoUrl, "helm-chart-repo-url", "https://redhat-developer.github.com/redhat-helm-charts", "Helm chart repository URL")
+	fs.StringVar(&cfg.repoUrl, "helm-chart-repo-url", "https://redhat-developer.github.io/redhat-helm-charts", "Helm chart repository URL")
```

Comment:
`https://redhat-developer.github.io/redhat-helm-charts/` is an HTML page. I'm guessing you don't want that as the default value here.

Reply:
we do. HTML page is for humans, but UI fetches https://redhat-developer.github.io/redhat-helm-charts/index.yaml

Reply:
We shouldn't proxy any HTML since we don't want to run any JS on the console domain. We should restrict the API to *only* proxy to the endpoint we want.